### PR TITLE
Fix blocking of evm RPC

### PR DESCRIPTION
**Pull request by tiannian**
* **make sure that you have executed all the following process and no errors occur**
  - [ ] make fmt
  - [ ] make lint
  - [ ] make test

* **The major changes of this PR**

* **The major impacts of this PR**
  - [ ] Impact WASM?
  - [ ] Impact Web3 API?
  - [ ] Impact mainnet data compatibility?

* **Extra documentations**

### CODE REVIEW

File: src/ledger/src/data_model/effects.rs [hunk 1]
```diff
@@ -360,7 +360,7 @@ impl TxnEffect {
 
         // Simplify (4)
         if !trn.body.lien_assignments.is_empty()
-            && trn.body.transfer_type != TransferType::Standard
+            || trn.body.transfer_type != TransferType::Standard
```
Comment (ws4charlie) [inline src/ledger/src/data_model/effects.rs:363]: I guess this is the fix already included in `v0.3.27-release`

Comment (tiannian) [reply src/ledger/src/data_model/effects.rs:363]: Already merge into develop, but main is unchange.

**Review by ws4charlie — APPROVED** [review]

**Comment by malefooo** [comment]
The following tests were done

- [x] no blocking
- [x] remix
    - [x] deploy
    - [x] call function
- [x] matemask
    - [x] transfer
- [x] hardhat
    - [x] deploy
- [x] fn 
    - [x] contrat-deposit

**Comment by ktmlm** [comment]
a rebase is needed for a more clear review

**Comment by tiannian** [comment]
This PR is hold, beacuse some features conflict with mainnet upgrade.

**Pull request merged**